src/web/app/components/question-types/question-statistics/rank-recipients-question-statistics.component.spec.ts

```diff
@@ -1,8 +1,17 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-
+import { FeedbackParticipantType, FeedbackRankRecipientsResponseDetails } from '../../../../types/api-output';
 import { SortableTableModule } from '../../sortable-table/sortable-table.module';
+import { Response } from './question-statistics';
 import { RankRecipientsQuestionStatisticsComponent } from './rank-recipients-question-statistics.component';
 
+/**
+ * Loads data for testing.
+ */
+const loadTestData: (filename: string) => Response<FeedbackRankRecipientsResponseDetails>[] =
+(filename: string): Response<FeedbackRankRecipientsResponseDetails>[] => {
+  return require(`./test-data/${filename}`);
```

Comment:
Any particular reason why test data is stored in a separate file? I'm not entirely against the approach don't get me wrong, just that our frontend tests so far all have their data embedded in the `spec.ts` file so I'm wondering if you had any particular reason for the extraction of data. 

I can understand the extraction if the same data is being used by other test components but this doesn't seem to be the case. Would be better for now to just have the data as a constant variable as in other frontend tests for consistency (e.g. `instructor-home-page.component.spec.ts`)

Reply:
Just felt that the code looked bad cosmetically with sample data and decided to put it in a JSON. Reverted

Reply:
You could consider putting up an issue for this actually, I remember it was something I wanted to raise but didn't have the time to. That is to say, refactor frontend test data into a dedicated resource folder for reusability instead of having it nested within the `spec.ts` files. We can have a community discussion about this, and if the idea is adopted start taking the approach as an epic/task.